app/services/wiki_service.rb

```diff
@@ -0,0 +1,35 @@
+class WikiService
+  include HTTParty
+  attr_reader :base_uri, :default_params
+
+  def initialize
+    @base_uri = "http://en.wikipedia.org/w/api.php"
+    @default_params = { action: "query",
+                        prop: "revisions",
+                        format: "json",
+                        rvprop: "content",
+                      }
+  end
+
+  def collect_data(titles)
+    default_params[:titles] = titles
+    summaries(responses).map { |summary| summary.split("\n")
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.